tools/logicalimager/tsk_logical_imager.cpp

```diff
@@ -392,28 +483,48 @@ main(int argc, char **argv1)
         usage();
     }
 
-    // If CWD is FAT, exit with error because it cannot create files greater 4 GB
-    if (DriveUtil::cwdIsFAT(cwd)) {
-        ReportUtil::consoleOutput(stderr, "Error: Writing to FAT device is not supported.\n");
-        ReportUtil::handleExit(1);
-    }
 
+    ////////////////////////////////////////////////////////
+    // Load the configuration file
     if (configFilename == NULL) {
         configFilename = _TSK_T("logical-imager-config.json");
         ReportUtil::consoleOutput(stdout, "Using default configuration file logical-imager-config.json\n");
     }
     ReportUtil::printDebug("Using config file %s", TskHelper::toNarrow(configFilename).c_str());
 
-    std::wstring wImgPathName;
-    std::vector<std::wstring> drivesToProcess;
+    try {
+        config = new LogicalImagerConfiguration(TskHelper::toNarrow(configFilename), (LogicalImagerRuleSet::matchCallback)matchCallback);
+        ReportUtil::SetPromptBeforeExit(config->getPromptBeforeExit());
+        createVHD = config->getCreateVHD();
+    }
+    catch (std::exception &e) {
+        std::cerr << e.what() << std::endl;
+        ReportUtil::handleExit(1);
+    }
+
+    // If CWD is FAT, exit with error because it cannot create files greater 4 GB
+    if (createVHD && DriveUtil::cwdIsFAT(cwd)) {
```

Comment:
Need to call cwdIsFAT because it outputs cwd. If non-VHD, a file can be greater than 4 GB.
if (DriveUtil::cwdIsFAT(cwd)) {

Reply:
No action needed. I have merged Brian's commit into my branch for 5503.

Reply:
@jkho, so is it correct that this PR can be closed?

Reply:
correct.